core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1552,6 +1553,13 @@ maybe_set_call_forward({Endpoint, Call, CallFwd, CCVs}) ->
                        )
     }.
 
+-spec maybe_rtcp_mux(ccv_acc()) -> ccv_acc().
+maybe_rtcp_mux({Endpoint, Call, CallFwd, CCVs} = Acc) ->
+    case kz_json:get_boolean_value([<<"media">>, <<"rtcp_mux">>], Endpoint, 'undefined') of
```

Comment:
Thanks you Luis @lazedo for the detailed explanation of why so and not otherwise
Also I added comments to FreeSwitch config to allow globaly disable/enable rtcp_mux without overloading rabbitmq

https://github.com/2600hz/kazoo-configs-freeswitch/pull/3